niceman/distributions/tests/test_singularity.py

```diff
@@ -46,3 +46,16 @@ def test_singularity_trace():
 
     # Clean up test files
     os.remove(image_file)
+
+    # Run tracer against a Singularity Hub url
+    files = ['shub:/GodloveD/lolcow']
```

Comment:
If this image is updated, this test will start to fail.  Pulling with the GitHub commit (appending "@a59d8de3121579fe9c95ab8af0297c2e3aefd827") would avoid that.